Classes/Person/person.py

```diff
@@ -0,0 +1,97 @@
+#Person
+import sys
+import random
+sys.path.append('../')
+import rooms
+
+class Person:
+    num_of_persons_at_dojo=0
+    persons=[]
+    rooms=[]
+    def __init__(self,rooms):
+        self.rooms=rooms
+        #print (self.rooms)
+
+    def add_person(self,name,p_type,accommodation="N"):
+        Person.num_of_persons_at_dojo+=1
+        self.name=name
+        self.p_type=p_type
+        self.accommodation=accommodation
+
+        x=self.name,self.p_type,accommodation
+        print(x[0])
+        if x[1]=="Staff"and x[2]=="Y":
+            return "Staff's Can't Get Accommodation"
+        
+        #self.persons.append(x[0])
+        #print (x[0]+" has been successfully added.")
+        #print (x[0]+" has been allocatedd ")
+        #print (self.persons)
+        pers=[]
+        pers.append(p_type)
+
+        #Add a person to room Randomly
+        if self.p_type=="Staff":
+            self.accommodation="N"
+            add_a_staffx=Staff().create_staff(name,self.rooms)
+            #Add Staff To Database/TextFile
+            print (add_a_staffx)
+            #return Room Allocated
+        
```

Comment:
should be `add_a_staffx` I admit my Variable names are poor. I'll rename them to be more descriptive